consensus/dummy/dynamic_fee_window.go

```diff
@@ -0,0 +1,88 @@
+// (c) 2025, Ava Labs, Inc. All rights reserved.
+// See the file LICENSE for licensing terms.
+
+package dummy
+
+import (
+	"encoding/binary"
+	"errors"
+	"fmt"
+
+	"github.com/ava-labs/avalanchego/utils/wrappers"
+	"github.com/ava-labs/coreth/params"
+	"github.com/ethereum/go-ethereum/common/math"
+)
+
+var ErrDynamicFeeWindowInsufficientLength = errors.New("insufficient length for dynamic fee window")
+
+// DynamicFeeWindow is a window of the last [params.RollupWindow] seconds of gas
+// usage.
+//
+// Index 0 is the oldest entry, and [params.RollupWindow]-1 is the current
+// entry.
+type DynamicFeeWindow [params.RollupWindow]uint64
+
+func ParseDynamicFeeWindow(bytes []byte) (DynamicFeeWindow, error) {
+	if len(bytes) < params.DynamicFeeExtraDataSize {
+		return DynamicFeeWindow{}, fmt.Errorf("%w: expected at least %d bytes but got %d bytes",
+			ErrDynamicFeeWindowInsufficientLength,
+			params.DynamicFeeExtraDataSize,
+			len(bytes),
+		)
+	}
+
+	var window DynamicFeeWindow
+	for i := range window {
+		offset := i * wrappers.LongLen
```

Comment:
(subjective) I think `wrappers.LongLen` adds very little while requiring the reader to check its value. We're extracting a `uint64` from bytes, so a raw `8` is an acceptable magic number IMO.

Reply:
Hmmm I feel we should keep using `wrappers.LongLen` because that aligns with the definition of:
```golang
const DynamicFeeExtraDataSize = RollupWindow * wrappers.LongLen
```